api/v1alpha1/kubernetes_helpers.go

```diff
@@ -24,6 +24,13 @@ func DefaultKubernetesDeploymentStrategy() *appv1.DeploymentStrategy {
 	}
 }
 
+// // DefaultKubernetesDaemonSetStrategy returns the default daemonset strategy settings.
```

Comment:
nit: double //

Reply:
Not double. DeploymentStrategy != DaemonSetUpdateStrategy